client/src/components/gameModeSettings/GameModesEditor.tsx

```diff
@@ -91,6 +92,12 @@ export default function GameModesEditor(props: Readonly<{
     
     
     return <div className="game-modes-editor">
+        <Helmet>
+            <meta name="twitter:site" content={translate("menu.start.title")}></meta>
+            <meta name="twitter:title" content={props.initialGameMode?.name}></meta>
+            <meta name="twitter:card" content="summary_large_image"></meta>
+            {/* <meta name="twitter:image:src" content="https://placehold.co/600x400"></meta> */}

```

Comment:
You should remove this commented out code

Reply:
This should be resolved now